planning/behavior_velocity_planner/autoware_behavior_velocity_blind_spot_module/src/util.cpp

```diff
@@ -681,4 +684,57 @@ std::optional<StopPoints> generate_stop_points(
     std::nullopt, stop_points_list.instant_stopline, stop_points_list.critical_stopline};
 }
 
+tl::expected<double, std::string> calc_ego_to_blind_spot_lanelet_lateral_gap(
```

Comment:
failure reason is almost obvious, so optional is enough for me

Reply:
done in https://github.com/autowarefoundation/autoware_universe/pull/11265/commits/894fcf90ae086af6c888b1b597b064023a7a70b1